### ISSUE TITLE
sitewide, forms 508-defect-3 [SCREENREADER]: Decorative content MUST be implemented in a way it can be ignored by assistive technology

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9169]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On the buttons there is a small arrow icon.  These icons should be hidden from screen readers, as they are decorative. This guidance applies to any instance of the arrow icon on the buttons throughout the product.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to  read the content of the page without extraneous noise from decorative images.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: all

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers/
2. Move through the form until you see the buttons with the arrows on them
3. Start screen reading device
4. Read content of the page
5. Verify that the arrows are read by the screen reader

## Possible Fixes (optional)

**Current code**
```html
<span class="button-icon"> »</span>
```

**Recommended code**
```html
<span aria-hidden="true" role="presentation" class="button-icon"> »</span>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG 1.1.1 - Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them](https://www.w3.org/TR/WCAG20-TECHS/F38.html)

* [W3C Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

## Screenshots or Trace Logs

![Screen Shot 2020-05-15 at 1 24 37 PM](https://user-images.githubusercontent.com/57469/82078641-77b4c400-96af-11ea-9906-12195bb9cfaf.png)

**Comment by jonathanjnelson** [comment]
@jenstrickland Feedback was integrated into staging environment. Can you confirm and close?

**Comment by jenstrickland** [comment]
The angle icons in the Back and Continue buttons are still read aloud in JAWS and NVDA. This is a sitewide issue, and assigned to VSP to address.
